tests/utils.py

```diff
@@ -28,12 +32,16 @@
 # Set during setUpSuite()
 _output_dir = ''
 
+# JS execution server and port
+_js_harness_port = ''
+
 
 def setUpSuite():
```

Comment:
At column 1: (C901) 'setUpSuite' is too complex (11)

Reply:
Brutal, man. It was already pretty complex. :P